src/NHSD.BuyingCatalogue.Solutions.API/ViewModels/Solution/LearnMoreSection.cs

```diff
@@ -0,0 +1,13 @@
+using NHSD.BuyingCatalogue.Solutions.Contracts;
+
+namespace NHSD.BuyingCatalogue.Solutions.API.ViewModels.Solution
+{
+    public sealed class LearnMoreSection
+    {
+        public LearnMoreSection(ILearnMore learnMore) => Answers = new LearnMoreSectionAnswers(learnMore);
+
+        public LearnMoreSectionAnswers Answers { get; }
```

Comment:
Looks good - Just one suggestion on style here.

Properties tend to go at the top of a class, this should be the convention in the rest of the solution.

Reply:
Disagree on this one - constructors should come before properties and this should be the convention for the solution.

Reply:
I don't see any value changing the convention at this stage of the project. I would like to review code styles at a later date seen as we have new editions to the team. For now though I would be grateful if you could make this code style change.

I'm surprised the Resharper code clean up tool didn't format the code with properties at the top as we specifically altered it to adhere to the current convention.

Reply:
I don't use R# for code formatting because it has a tendency to make a mess of my code.

At what point was it agreed to deviate from the convention of having constructors before properties?

Reply:
Ah OK makes sense.

Since the start of the project.

Reply:
Ah, OK. However, given that no one style is currently applied consistently throughout the solution I'm not sure what value would be gained by changing the style here.

I would rather wait until any code style review has taken place and update after that if necessary, although I will be arguing for the style used here ;)

Reply:
I'm confused. This does mean you have seen areas of the application that don't adhere to properties at the top of the file? Because this convention should be applied throughout the source code.

Reply:
Yes, that's correct. There are definitely some classes that follow the style of constructors before properties. There is even at least one class that doesn't follow either style. I guess there may be more but that would involve checking each class or adding StyleCop (which would highlight the number of classes where properties come before constructors).

Reply:
The code style convention should have been applied across the source code. The odd defect should not mean we add a new style as that will lead to more confusion. So I would be grateful if we could stick to the current code style convention of properties above a constructor. At least we have a record of your disagreement, which we can refer to at a later date.